### PR TITLE
Avalon 7.4.0 styling

### LINKED ISSUE #3: Avalon: Update header and footer

### PR DESCRIPTION
**Pull request by qtamu**
Fixed broken Tamu Styling for Avalon 7.4.0

### CODE REVIEW

File: app/assets/stylesheets/tamu.scss [hunk 1]
```diff
@@ -0,0 +1,74 @@
+#header-navbar .navbar-brand {
+    background: transparent "tamu-logo-with-bar.png" no-repeat top left;
+    // background: transparent url(/assets/tamu/tamu-logo-with-bar.png) no-repeat top left;
+	margin-top: 10px;
+    margin-bottom: -10px;
+    
+}
+
```
Comment (kaladay) [inline app/assets/stylesheets/tamu.scss]: The tabbing is inconsistent.

I looked at the other SCSS files and I am seeing 2-space tabbing.
Can you update this entire file to consistently use 2-space tabbing?

File: app/views/modules/_header.html.erb [hunk 2]
```diff
@@ -13,14 +13,27 @@ Unless required by applicable law or agreed to in writing, software distributed
...
+					<a href="http://askus.library.tamu.edu">Help</a>
+				</li>
+			</ul>
+		</div>
+	  </div>
+
     </header>
   </div>
-</div>
+</div>
```
Comment (kaladay) [inline app/views/modules/_header.html.erb]:
```suggestion
</div>

```

Add missing EOL at the EOF.

**Review by kaladay — CHANGES_REQUESTED** [review]

File: app/assets/stylesheets/tamu.scss (hunk 1 above)
Comment (qtamu) [reply app/assets/stylesheets/tamu.scss]: done

File: app/assets/stylesheets/tamu.scss [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+  margin-top: 10px;
+  margin-bottom: -10px;
+}
+
+#header-navbar .navbar-toggle, #header-navbar .facets-toggle {
+  background: #ffcf08;
+}
+
+.container > .navbar-header, .container > .navbar-collapse, .container > #facet-panel-collapse, .container > div[id*='facet-panel-'], .container-fluid > .navbar-header, .container-fluid > .navbar-collapse, .container-fluid > #facet-panel-collapse, .container-fluid > div[id*='facet-panel-'] {
+  background-color: #500000;
```
Comment (tamu-sad-iii) [inline app/assets/stylesheets/tamu.scss]: Can we update this scss file to import the bootstrap/branding theme variables if necessary and use theme where applicable? Also, we should be able to remove overrides of bootstrap classes in which the same thing the variable would apply by default.

**Review by kaladay — APPROVED** [review]

**Pull request merged**